system_tests/monitoring.py

```diff
@@ -17,10 +17,15 @@
 from gcloud import _helpers
 from gcloud.environment_vars import TESTS_PROJECT
 from gcloud.exceptions import NotFound
+from gcloud.exceptions import ServiceUnavailable
 from gcloud import monitoring
 
+from retry import RetryErrors
 from system_test_utils import unique_resource_id
 
+retry_404 = RetryErrors(NotFound)
+retry_503 = RetryErrors(ServiceUnavailable)
```

Comment:
Sorry I didn't mention it on the other, can you capitalize since these are globals?